README.md

```diff
@@ -192,9 +198,15 @@ To deploy the service in a production environment, follow these steps:
 
 2. **Configure Environment Variables**
    - Create a `.env` file in the deployment directory.
-   - Add your OpenAI API key like this:
+   - Obtain all the relevant secrets for the deployment:
```

Comment:
Can you add some links to docs where one can get all of them?

Reply:
Yes - in this PR or as a separate pull request that then fulfills #38 ?